scripts/setup-dev.js

```diff
@@ -28,22 +35,23 @@ const cmsDataPath = path.resolve(__dirname, '..', 'public', 'data', 'cms')
 
     if (!vamcEhrExists) {
       // Grab data file populated by the cms
-      await fetch('https://va.gov/data/cms/vamc-ehr.json')
-        .then((res) => res.json())
-        .then((data) => {
-          fs.mkdirp(cmsDataPath)
-            .then(() => {
-              fs.writeJson(`${cmsDataPath}/vamc-ehr.json`, data)
-              // eslint-disable-next-line no-console
-              console.log('vamc-ehr data fetched successfully!')
-            })
-            .catch((err) => {
-              console.error('Error with cms data directory: ', err)
-            })
-        })
-        .catch((err) => {
+      const data = await fetch('https://va.gov/data/cms/vamc-ehr.json').catch(
+        (err) => {
           console.error('Error fetching cms data from va.gov: ', err)
-        })
+        }
+      )
+      // eslint-disable-next-line no-console
+      console.log('vamc-ehr data fetched successfully!')
+
+      await fs
+        .mkdirp(cmsDataPath)
+        .catch((err) =>
+          console.error('Error creating cms data directory: ', err)
+        )
+
+      fs.writeJson(`${cmsDataPath}/vamc-ehr.json`, data).catch((err) =>
+        console.error('Error writing data file: ', err)
+      )
```

Comment:
This looks better (and, again, will work), but, at least as I understand things, it's still not idiomatic/best practice. Generally, `await` is used within `try...catch` blocks in order to function more like synchronous code. This is still mixing that syntax with the more traditional `Promise` handling that uses chaining.

Two possible fixes:
1. Remove `await` in the first iteration and move on. That is probably the easiest fix, but probably not the best. We are using `await` fairly consistently in this project, and I think this code is best implemented to match that convention.
2. 
```
let data
try {
  data = await fetch('https://va.gov/data/cms/vamc-ehr.json')
} catch(err) {
  console.error('Error fetching cms data from va.gov: ', err)
}

// eslint-disable-next-line no-console
console.log('vamc-ehr data fetched successfully!')

try {
  await fs.mkdirp(cmsDataPath)
} catch(err) {
  console.error('Error creating cms data directory: ', err)
}

try {
  await fs.writeJson(`${cmsDataPath}/vamc-ehr.json`, data)
} catch(err) {
  console.error('Error writing data file: ', err)
)
```

As I type that, though, I recognize that the calls are dependent on one another, and those dependencies seem to be covered appropriately in the `Promise` chain approach. The endless indentation is hard on the eyes, and one big benefit of this approach using `await`, but we'd need to exit appropriately if calls fail.